Бот {recommendation} {service}

Slobodskoy opened the pull request.

Slobodskoy committed:
Бот {recommendation} {service}

LiFeAiR commented on internal/app/commands/recommendation/commander.go:
```diff
@@ -0,0 +1,47 @@
+
+func (c *RecommendationCommander) HandleCallback(callback *tgbotapi.CallbackQuery, callbackPath path.CallbackPath) {
+	switch callbackPath.Subdomain {
+	case "subdomain":
```
`service` же

Slobodskoy replied:
Исправил

LiFeAiR commented on internal/app/commands/recommendation/service/commander.go:
```diff
@@ -0,0 +1,49 @@
+
+func (c *RecommendationServiceCommander) HandleCommand(msg *tgbotapi.Message, commandPath path.CommandPath) {
+	switch commandPath.CommandName {
+	case "help":
```
а где `create`?

Slobodskoy replied:
Добавил

LiFeAiR commented on internal/service/recommendation/service/models.go:
```diff
@@ -0,0 +1,24 @@
+}
+
+type Service struct {
+	Id    uint64
```
у сущности должны быть такие поля

- id
- title
- description
- rating

Slobodskoy replied:
Исправил

LiFeAiR commented on internal/service/recommendation/service/service.go:
```diff
@@ -0,0 +1,78 @@
+func NewDummyServiceService(allEntities []Service) *DummyServiceService {
+	data := make(map[uint64]*Service, len(allEntities))
+	var lastId uint64 = 0
+	for _, entity := range allEntities {
```
зачем заполнять фейковыми данными?

Slobodskoy replied:
Что бы бот хоть с чем то запускался и можно было сразу проверить list и delete

Slobodskoy committed:
Исправления по ревью
1. Добавлены поля сущности
2. Добавлен роут и функционал добавления сущности
3. Правки ошибок
